platform/jvm/capture/src/main/kotlin/io/bitdrift/capture/LoggerImpl.kt

```diff
@@ -172,6 +173,7 @@ internal class LoggerImpl(
                 eventsListenerTarget,
                 clientAttributes.appId,
                 clientAttributes.appVersion,
+                Build.MODEL,
```

Comment:
This is already abstracted via `DeviceAttributes` we should expose it there

Reply:
OK I will change this around.